WPMedia/fusion-news-theme-blocks
# THEME-738: Identity | Account Management

## Description

This PR converts the account management block to Themes v2. There's some differences between the mock ups and these changes. Notably, there's no display name functionality yet (as it was not in the v1 block) and the social section keeps the functionality as it was in v1. There will be follow up tickets to make these parts match the mock ups at a later date.

## Jira Ticket

- [THEMES-738](https://arcpublishing.atlassian.net/browse/THEMES-738)

## Acceptance Criteria

- The account management block is converted to v2.

## Test Steps

1. Checkout this branch `git checkout THEMES-738`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. The account management page requires a logged in user to view.
4. Create two separate pages in PageBuilder, one with the 'Identity Log In - Arc Block' and another with the 'Identity Account Management – Arc Block'.
5. Share and publish both.
6. Navigate to the login page you created and use the test login for The Gazette listed here: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2794422817/QA.
7. After logging in, go to the account management page and verify you can see the user info populated.
8. Make sure the editable fields (email and password) can be toggled, but do not actually modify the test account's email or password.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-738]: https://arcpublishing.atlassian.net/browse/THEMES-738?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
